spec/lib/search_spec.rb

```diff
@@ -2541,6 +2541,23 @@ def indexed_post(*args)
     end
   end
 
+  describe "include:invisible / include:unlisted" do
+    it "allows including invisible topics in the results" do
+      topic = Fabricate(:topic, title: "I am testing a search", visible: false)
+      _post2 = Fabricate(:post, topic: topic, raw: "this is the second post", post_number: 2)
+      post = Fabricate(:post, topic: topic, raw: "this is the first post", post_number: 1)
```

Comment:
Nitpicking but I would swap these two lines to keep them "in order" of their `post_number`.

```suggestion
      post = Fabricate(:post, topic: topic, raw: "this is the first post", post_number: 1)
      _post2 = Fabricate(:post, topic: topic, raw: "this is the second post", post_number: 2)
```